[CS2113T-T12-2] 25HoursADay

Jingming517 opened the pull request:
Product name: 25HoursADay
Target User profile: Students (NUS students)
Value Proposition:
Easy usage to-do-list (user is able to key in pre-set abbreviation to indicate different types of tasks)
Helps student prioritize their tasks
Able to import information from NUS websites, without the need to type out the task one by one

abnermtj commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+### Architecture
+
+The figure below shows the overall design of the application. Given below is a sequence diagram when adding a Todo task.
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/Overall_Architecture.JPG" alt="" width="300"/> <br/>
```
Perhaps a shift the sequence diagram reference to after this image, as I was confused for a while, thinking its the figure being referenced.

abnermtj commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+
+UI: The user interface of the application.
+Parser: Interprets the user's input.
+Command: The command executor.
```
perhaps add a new line character to the end of these lines. They come out in different formatting from what I assume you intended
![image](https://user-images.githubusercontent.com/55742254/97387099-499b0780-1910-11eb-87bc-13d639275e23.png)

abnermtj commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+Model: Holds the data of the application in memory.
+Storage: Reads data from, and writes data to, the hard disk. 
+
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/Archi_SD.JPG" alt="" width="750"/>
```
![image](https://user-images.githubusercontent.com/55742254/97387798-a814b580-1911-11eb-8352-6b07c4387261.png)

Great job!
I think it can be improved by
1) not certain if having the activation bar directly side by side to object will imply the operation below happen in the constructor of main? It doesn't seem to be the case.

abnermtj commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+Storage: Reads data from, and writes data to, the hard disk. 
+
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/Archi_SD.JPG" alt="" width="750"/>
+
```
2. Maybe this storage activation bar can be shift up since          storage.writeToFile(calendarList) returns before the add comannd ends

abnermtj commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+### Add a calendar item feature
+This feature is facilitated by `AddCommand`.
+The following sequence diagram shows how the `execute()` operation works:
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/add_command_SD.JPG" alt="" width="750"/><br/>
```
The uml diagram here is slightly inconsistent with the ones below. Perhaps in a future version can startdardize the if using the alt box and object deletion parts

abnermtj commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+The following sequence diagram show how the `execute()` operation works:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/deleteCommand_SD.JPG" alt="" width="750"/><br/>
+
+Note: It first extracts the task/event number from the user input prior to `convertTaskNumberToCalendarNumber` and `convertEventNumberToCalendarNumber` respectively. 
```
There are light formatting errors in the diagrams here. Perhaps the life line needs to connect to the center of the object deletion cross. Some method calls are crossing the box boundaries.

abnermtj commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/deleteCommand_SD.JPG" alt="" width="750"/><br/>
+
+Note: It first extracts the task/event number from the user input prior to `convertTaskNumberToCalendarNumber` and `convertEventNumberToCalendarNumber` respectively. 
+
```
I believe that the middle of the box alt path should be a dotted line

abnermtj reviewed (approved):
Overall very good developer guide!

thngyuxuan commented on src/main/java/seedu/duke/calendar/CalendarList.java:
```diff
@@ -0,0 +1,213 @@
+     *
+     * @param task to be added.
+     */
+    public void addTask(Task task) {
```
is there supposed to be a function called to the Todo class? I am not sure how the Todo class is activated when addTask is called
![image](https://user-images.githubusercontent.com/60374106/97389075-82d57680-1914-11eb-924f-42e510412861.png)

thngyuxuan commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+This feature is facilitated by `AddCommand`.
+The following sequence diagram shows how the `execute()` operation works:
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/add_command_SD.JPG" alt="" width="750"/><br/>
+
```
I think there are supposed to be some return lines to denote how/why the activation bar ends! (for CalendarList, UI, and Storage in the UML diagram)

thngyuxuan commented on docs/DeveloperGuide.md:
```diff
@@ -1,33 +1,372 @@
+### Print tasks feature
+This feature is facilitated by `PrintTasksCommand`.
+The following sequence diagram shows how the `execute()` operation works when the user wants to print the list of tasks stored in the program.
+![print_tasks_command_sd](images/PrintTasksCommand_SD.png)
```
I think there's a formatting error on your UML diagrams, but the dotted line needs to extend to the centre of the deletion cross!

thngyuxuan reviewed (commented):
Overall, it is a nicely written DG! The introduction and setting up guide was very detailed and easy to follow. 
The UML diagrams had similar formatting errors so it might be due to rendering issues that caused the same errors, other than that everything else is fine :)

thesamchris reviewed (commented):
Hello guys! Good job on your TP: 25 hours a day.

Here are a few comments:
1. The name of the PR should be 2113T instead of 2113 cause when we click the PR button from the table of allocation for peer review, there is nothing that shows up.
2. [Optional] For the example diagram in the design section, I think there should be a return arrow for the addTodoTask()
![image](https://user-images.githubusercontent.com/13195220/97513674-d8b72680-19c7-11eb-851c-ae8015be5db4.png)
3. For the storage portion, I feel that a good detail to elaborate on is how exactly does "25 Hours In A Day" store the data, is it through a text file? If so, an interesting insight for readers might be the way the data is structured. Also currently I notice that the data is stored after each command is run, so would be good to explain.
![image](https://user-images.githubusercontent.com/13195220/97513991-a3f79f00-19c8-11eb-8b93-60764f8300d2.png)
4. [Optional] I noticed this in a few of the sequence diagrams, I think there should be a return arrow for the functions that are called.
![image](https://user-images.githubusercontent.com/13195220/97514277-49ab0e00-19c9-11eb-9403-2503ada26cde.png)
5. To improve the ease of finding things on the DG, maybe we can consider to split the Implementation portion into different parts dealing with the different entities? Like Calendar event commands, Todo commands, etc?
6. Overall, I do not see screenshots included in the DG apart from the welcome screen. I think it would make it even better if we get to see the application's output in the DG. For example, commands like "print countdown".
7. This diagram from countdown command looks cluttered, maybe we can space things out a bit more? Also the rectangles should not be rounded and is there supposed to be an "X" on the UI object? Normally the UI component is there throughout the application.
![image](https://user-images.githubusercontent.com/13195220/97515138-53ce0c00-19cb-11eb-9065-8edcd9a637d3.png)

lhydl commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+|[Section 4: Implementation](#implementation)|To explain the mechanism of functions added to *25HoursADay* and the features added to *25HoursADay*.|
+|[Section 5: Documentation](#documentation)|To represent documents describing the system and its parts.|
+
+## Setting Up & Getting started
```
Perhaps can label the main headings with the section numbers for better readability.

lhydl commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+        1. Click `Configure` → `Project Structure for New Project`
+        1. Click `New...` and set it to the directory of the JDK
+1. Upon the successful configuration, you should be able to see the following screenshot:
+![JDK Config](images/JDK_Config_Screenshot.png)
```
I like how you insert the screenshot of the configuration here so that the readers can confirm that they configure the ide correctly.

EyoWeiChin commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+
+The different `Command` classes are responsible for receiving the user's input from the `Parser` class and executing the corresponding commands. 
+The figure belows shows the class diagram of the command class: <br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/command.JPG" alt="" width="200"/><br/>
```
![image](https://user-images.githubusercontent.com/18654251/97659384-d5946700-1aa9-11eb-85ce-9d3a31b0d77c.png)

Perhaps you can expand this class diagram by showing how each Command class will interact or inherit from the abstract Command class.

EyoWeiChin commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+#### Add additional information
+This feature is facilitated by `AddInfoCommand`.
+The following sequence diagram show how the `execute()` operation of `AddInfoCommand` works:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/addInfoCommand_SD.JPG" alt="" width="750"/><br/>
```
![image](https://user-images.githubusercontent.com/18654251/97659672-826ee400-1aaa-11eb-8fa7-5e3d7632c83f.png)

There seems to an inconsistent format with :CalendarList having a dotted line

lhydl replied:
![Screenshot 2020-10-30 122323](https://user-images.githubusercontent.com/13282967/97660574-fca06800-1aac-11eb-8783-b133da00126c.png)

Are the 'x's surrounding the CalendarList class intended? This seems inconsistent with all other diagrams. And I believe that the lifeline should touch the deletion X.

lhydl commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+### Add a calendar item feature
+This feature is facilitated by `AddCommand`.
+The following sequence diagram shows how the `execute()` operation works:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/add_command_SD.JPG" alt="" width="750"/><br/>
```
![Screenshot 2020-10-30 124101](https://user-images.githubusercontent.com/13282967/97660665-383b3200-1aad-11eb-8ac0-2c24dea46db6.png)
I think the line used to separate the alt frame has to be a dotted line instead of a solid line.

EyoWeiChin commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+### Delete a calendar item feature
+This feature is facilitated by `DeleteCommand`.
+The following sequence diagram show how the `execute()` operation works:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/deleteCommand_SD.JPG" alt="" width="750"/><br/>
```
![image](https://user-images.githubusercontent.com/18654251/97659815-eb565c00-1aaa-11eb-8a66-caf54dd12fd2.png)
Perhaps it would be more accurate if remove method call to the Task object be linked to an activation bar instead of directly to the object termination. The object can then be terminated soon after.

Also, it seems that the activation bar for Calendarlist is being overlapped by the objects lifeline.

lhydl commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+Similarly, it uses`getFirstTodo()` function to get the first added todo task in the list, and `getFirImportantTodo()` function to get the first added important todo task in the list.
+If no corresponding items, the functions will return null.
+
+### Print countdown feature
```
I feel like for all of the implementations, can explain the implementations briefly in words first before inserting the sequence diagrams for further illustration. I feel like this may allow readers to better understand the implementations.

lhydl commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+This feature is facilitated by `CountdownCommand`.
+
+The following sequence diagram shows how the `execute()` operation works when the user decide to see the countdown of exams or deadlines:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/countdown_command_SD.jpg" alt="" width="750"/><br/>
```
Really good colour coordination of the classes and packages for the diagrams. Diagrams are also generally consistent. Good job!

EyoWeiChin commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+### Print suggestions feature
+This feature is facilitated by `PrintSuggestionCommand`.
+The following sequence diagram shows how the `execute()` operation works when the user wants to see suggestions about preparing which tasks:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/PrintSuggestionCommand_SD.png" alt="" width="500"/><br/>
```
![image](https://user-images.githubusercontent.com/18654251/97660308-19886b80-1aac-11eb-9c95-d02bb1a22bee.png)
This diagram is a really good effort, however, it might be too complicated, as there is a lot of things going on. Perhaps, it would be beneficial to split it up into smaller parts, or focus on an individual aspect!

EyoWeiChin commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+The following sequence diagram further illustrates the above process. 
+![ModuleChecker_SD]()
+
+## Documentation
```
Good documentations!

EyoWeiChin commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,503 @@
+### Print prioritized tasks feature
+This feature is facilitated by `PrintPriorityCommand`.
+The following sequence diagram shows how the `execute()` operation works when the user wants to print all the prioritized tasks:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/PrintPriorityCommand_SD.png" alt="" width="750"/><br/>
```
![image](https://user-images.githubusercontent.com/18654251/97660644-2eb1ca00-1aad-11eb-93d3-dabca535d023.png)

Since the else field seems empty (unless there is a self-invocation, perhaps it would be better to use alt instead?
Also, it seems that the printImportantTask() arrow is not properly formatted

EyoWeiChin reviewed (commented):
Overall a well-written and documented guide, aside from some minor formatting issues! I hope you will find my comments useful!

imhm committed:
Update Add Command feature in DG

Jingming517 committed:
Merge pull request #298 from Jingming517/master

Added label in DG

imhm committed:
Code refactoring

imhm committed:
Merge pull request #297 from imhm/fix-add-info-bug

Update Find Command parsing User input

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

imhm committed:
Merge branch 'master' into developer-guide-branch

# Conflicts:
#	docs/images/tip_find.PNG

imhm committed:
Merge pull request #299 from imhm/developer-guide-branch

Developer guide branch

LIU-YiFeng-1 committed:
TOC?

LIU-YiFeng-1 committed:
toc2

LIU-YiFeng-1 committed:
TOC

LIU-YiFeng-1 committed:
Merge pull request #300 from LIU-YiFeng-1/working_for_TP

test toc

LIU-YiFeng-1 committed:
TOC

LIU-YiFeng-1 committed:
Merge pull request #301 from LIU-YiFeng-1/working_for_TP

TOC

LIU-YiFeng-1 committed:
toc ?

LIU-YiFeng-1 committed:
Merge pull request #302 from LIU-YiFeng-1/working_for_TP

toc ?

Jingming517 committed:
Table of contents for DG

Jingming517 committed:
UG

LIU-YiFeng-1 committed:
update UG sequence

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

LIU-YiFeng-1 committed:
Merge pull request #303 from LIU-YiFeng-1/working_for_TP

update UG sequence

Jingming517 committed:
Merge pull request #304 from Jingming517/master

Table of contents for DG

Zhangyilin0203 committed:
change SD of countdown

Zhangyilin0203 committed:
modify code in countdown class

LIU-YiFeng-1 committed:
update SD

LIU-YiFeng-1 committed:
Merge pull request #305 from LIU-YiFeng-1/working_for_TP

update SD

Jingming517 committed:
Changes to DG

Jingming517 committed:
Merge pull request #306 from Jingming517/master

Changes to DG

Zhangyilin0203 committed:
update SD and DG

Zhangyilin0203 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into change-countdown

LIU-YiFeng-1 committed:
Drafting PPP

LIU-YiFeng-1 committed:
update PPP link

Zhangyilin0203 committed:
Merge pull request #307 from Zhangyilin0203/change-countdown

Improve SD in DG

JiawenLyu committed:
Added PrintPriorityCommand_SD.png

JiawenLyu committed:
Added PrioritizeCommand_SD.png

JiawenLyu committed:
Added PrintSuggestionCommand_SD.png

JiawenLyu committed:
Added PrintProgressCommand_SD.png

Zhangyilin0203 committed:
Add countdown's manual testing

JiawenLyu committed:
Added PrintProgressCommand_SD.png

JiawenLyu committed:
Merge pull request #308 from JiawenLyu/master

Modify SDs.

JiawenLyu committed:
Merge branch 'master' of https://github.com/JiawenLyu/tp

Zhangyilin0203 committed:
Merge pull request #309 from Zhangyilin0203/manual-testing

Add countdown feature for manual testing

JiawenLyu committed:
Add result for print suggestion in DG.

JiawenLyu committed:
Merge remote-tracking branch 'upstream/master'

JiawenLyu committed:
Update DG.

JiawenLyu committed:
Merge pull request #310 from JiawenLyu/master

Update DG.

LIU-YiFeng-1 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into working_for_TP

LIU-YiFeng-1 committed:
update DG with corrected verifu_setup.PNG

LIU-YiFeng-1 committed:
Standardized image package

LIU-YiFeng-1 committed:
Merge pull request #311 from LIU-YiFeng-1/working_for_TP

refactor image package

LIU-YiFeng-1 committed:
update PPP

LIU-YiFeng-1 committed:
update PPP

LIU-YiFeng-1 committed:
update code reference link

LIU-YiFeng-1 committed:
update code quality, removed unwanted comments

LIU-YiFeng-1 committed:
Merge pull request #312 from LIU-YiFeng-1/working_for_TP

update PPP and fix some code comments

LIU-YiFeng-1 committed:
update Junit for ModuleChekcer

Zhangyilin0203 committed:
Avoid too long method

Zhangyilin0203 committed:
Delete unused method

Zhangyilin0203 committed:
Update ppp

JiawenLyu committed:
Update prioritize command if the task is already prioritized.

JiawenLyu committed:
Modify prioritize function for the invalid input.

JiawenLyu committed:
Merge branch

JiawenLyu committed:
Merge remote-tracking branch 'upstream/master'

JiawenLyu committed:
Update prioritize command

JiawenLyu committed:
Merge pull request #313 from JiawenLyu/master

Modify prioritize command.

Zhangyilin0203 committed:
Improve storage

Zhangyilin0203 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into I-codequality

LIU-YiFeng-1 committed:
update some Junit test

LIU-YiFeng-1 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into working_for_TP

Zhangyilin0203 committed:
Update PPP

Zhangyilin0203 committed:
Merge pull request #314 from Zhangyilin0203/I-codequality

Improve code quality and storage

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

LIU-YiFeng-1 committed:
Update UG with the following:
1. command parameters are case-sensitive
2. command parameters are not to be used as placeholders
3. added the phrase "Note: You can input an activity description with or without spaces." to the add activity section
4. added the phrase "Note: You can input an task description with or without spaces." to the add task section

LIU-YiFeng-1 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into working_for_TP

LIU-YiFeng-1 committed:
Merge pull request #315 from LIU-YiFeng-1/working_for_TP

update UG

Zhangyilin0203 committed:
Update SD

LIU-YiFeng-1 committed:
update rationale for how items are printed in `print events`, `print tasks` and find function.

LIU-YiFeng-1 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into working_for_TP

LIU-YiFeng-1 committed:
Merge pull request #316 from Zhangyilin0203/I-SD

Update SD

LIU-YiFeng-1 committed:
Merge pull request #317 from LIU-YiFeng-1/working_for_TP

update UG

Zhangyilin0203 committed:
Update Load data SD

LIU-YiFeng-1 committed:
update figure in UG

LIU-YiFeng-1 committed:
Merge pull request #319 from LIU-YiFeng-1/working_for_TP

update figure in UG

Zhangyilin0203 committed:
Merge pull request #318 from Zhangyilin0203/I-SD

Update Load data SD

imhm committed:
update SD bugs

imhm committed:
Merge branch 'master' into developer-guide-branch

imhm committed:
Merge pull request #320 from imhm/developer-guide-branch

Update SD bugs

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

Jingming517 committed:
Changed timeline SD in DG

LIU-YiFeng-1 committed:
update error messgae for activity

LIU-YiFeng-1 committed:
update error message for user

LIU-YiFeng-1 committed:
Merge pull request #322 from LIU-YiFeng-1/working_for_TP

update error message

LIU-YiFeng-1 committed:
Merge pull request #321 from Jingming517/master

Changed timeline SD in DG

LIU-YiFeng-1 committed:
update PPP

LIU-YiFeng-1 committed:
update PPP

LIU-YiFeng-1 committed:
Merge pull request #323 from LIU-YiFeng-1/working_for_TP

update PPP

imhm committed:
Fix bug in user input parsing

imhm committed:
Merge pull request #324 from imhm/fix-add-info-bug

Fix bug in user input parsing

Zhangyilin0203 committed:
Update SD

LIU-YiFeng-1 committed:
update UG with addtional notes on unnessary spaces

imhm committed:
update UG legend pic

LIU-YiFeng-1 committed:
Merge pull request #326 from LIU-YiFeng-1/working_for_TP

update UG with addtional notes on unnessary spaces

LIU-YiFeng-1 committed:
Merge pull request #325 from Zhangyilin0203/A-SD

Update SD

imhm committed:
Merge pull request #327 from imhm/fix-add-info-bug

update UG legend pic

LIU-YiFeng-1 committed:
udpate typo

LIU-YiFeng-1 committed:
Merge pull request #328 from LIU-YiFeng-1/working_for_TP

udpate typo

LIU-YiFeng-1 committed:
update TOC link

JiawenLyu committed:
Update toc

LIU-YiFeng-1 committed:
fix TOC link

LIU-YiFeng-1 committed:
Merge pull request #330 from LIU-YiFeng-1/working_for_TP

Working for tp

JiawenLyu committed:
Merge branch 'master' into master

LIU-YiFeng-1 committed:
update module checker message

LIU-YiFeng-1 committed:
Merge pull request #331 from LIU-YiFeng-1/working_for_TP

update module checker message

LIU-YiFeng-1 committed:
no message

LIU-YiFeng-1 committed:
Merge pull request #332 from LIU-YiFeng-1/working_for_TP

no message

LIU-YiFeng-1 committed:
update notes for spacing of the command parameter

LIU-YiFeng-1 committed:
fix typo in print timeline date <date>

JiawenLyu committed:
Merge pull request #329 from JiawenLyu/master

Update toc

LIU-YiFeng-1 committed:
update chat box

LIU-YiFeng-1 committed:
fix bug in Module Checker and its exception

LIU-YiFeng-1 committed:
Merge pull request #333 from LIU-YiFeng-1/working_for_TP

update typo and chat box

LIU-YiFeng-1 committed:
FIX CHECK STYLE

LIU-YiFeng-1 committed:
Merge pull request #334 from LIU-YiFeng-1/working_for_TP

FIX CHECK STYLE

LIU-YiFeng-1 committed:
fix checks

LIU-YiFeng-1 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into working_for_TP

# Conflicts:
#	data/tasks.txt

JiawenLyu committed:
Merge branch

JiawenLyu committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	data/tasks.txt
Confirm merge

LIU-YiFeng-1 committed:
fix checks

LIU-YiFeng-1 committed:
Merge pull request #335 from LIU-YiFeng-1/working_for_TP

Working for tp

Zhangyilin0203 committed:
Add author

Zhangyilin0203 committed:
Merge pull request #336 from Zhangyilin0203/author

Improve code

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

Jingming517 committed:
Fixed past date bug in timeline

Jingming517 committed:
Merge pull request #337 from Jingming517/master

Fixed past date bug in timeline

Zhangyilin0203 committed:
Improve Ui

JiawenLyu committed:
Throw new exception for empty venue.

JiawenLyu committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	data/tasks.txt
Confirm merge

JiawenLyu committed:
Fix checkstyle

Jingming517 committed:
Updated PPP

imhm committed:
Update hong ming PPP
Add new error message if additional information is empty

Zhangyilin0203 committed:
Merge pull request #338 from Zhangyilin0203/bugs-fix

Improve Ui

LIU-YiFeng-1 committed:
fix PPP

LIU-YiFeng-1 committed:
Merge pull request #341 from LIU-YiFeng-1/working_for_TP

fix PPP

JiawenLyu committed:
Improve code quality

Jingming517 committed:
Updated PPP

imhm committed:
Code refactoring

JiawenLyu committed:
Merge pull request #339 from JiawenLyu/master

Deal with empty venue

Zhangyilin0203 committed:
fix output bug in storage

imhm committed:
Merge pull request #340 from imhm/final-bug-fixes

Update hong ming's PPP

LIU-YiFeng-1 committed:
Merge pull request #342 from Jingming517/master

Updated PPP

imhm committed:
Update AboutUs.md

LIU-YiFeng-1 committed:
Merge pull request #343 from Zhangyilin0203/bugs-fix

fix output bug in storage

LIU-YiFeng-1 committed:
update theme

LIU-YiFeng-1 committed:
Merge pull request #344 from LIU-YiFeng-1/working_for_TP

update theme

Zhangyilin0203 committed:
fix bug in storage

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

Zhangyilin0203 committed:
Merge pull request #345 from Zhangyilin0203/bugs-fix

fix bug in storage

imhm committed:
update find exception message

imhm committed:
Merge branch 'master' into final-bug-fixes

LIU-YiFeng-1 committed:
back to grey theme

LIU-YiFeng-1 committed:
udpate space

Jingming517 committed:
Updated PPP

LIU-YiFeng-1 committed:
back to correct version

Jingming517 committed:
Merge pull request #346 from Jingming517/master

Updated PPP

Zhangyilin0203 committed:
Remove table of content

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

LIU-YiFeng-1 committed:
remove TOC

Jingming517 committed:
Removed toc in PPP

Zhangyilin0203 committed:
Merge pull request #347 from Zhangyilin0203/bugs-fix

Remove table of content

LIU-YiFeng-1 committed:
Merge pull request #349 from LIU-YiFeng-1/working_for_TP

Working for tp

Jingming517 committed:
Merge pull request #348 from Jingming517/master

Removed toc in PPP

JiawenLyu committed:
Delete Toc

LIU-YiFeng-1 committed:
update PPP

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

Zhangyilin0203 committed:
fixing PPP

LIU-YiFeng-1 committed:
Merge pull request #350 from LIU-YiFeng-1/working_for_TP

update PPP

imhm committed:
update PPP

JiawenLyu committed:
Modify PPP

LIU-YiFeng-1 committed:
update PPP

imhm committed:
Merge pull request #352 from imhm/final-bug-fixes

Final bug fixes

Zhangyilin0203 committed:
Merge pull request #351 from Zhangyilin0203/bugs-fix

fixing PPP

JiawenLyu committed:
Update PPP

LIU-YiFeng-1 committed:
update PPP

LIU-YiFeng-1 committed:
Merge pull request #354 from LIU-YiFeng-1/working_for_TP

Working for tp

imhm committed:
update ppp to fit 2 pages

imhm committed:
Merge pull request #355 from imhm/final-bug-fixes

update ppp to fit 2 pages

JiawenLyu committed:
Merge pull request #353 from JiawenLyu/master

Update PPP

LIU-YiFeng-1 committed:
update PPP

LIU-YiFeng-1 committed:
Merge pull request #356 from LIU-YiFeng-1/working_for_TP

update PPP

JiawenLyu committed:
Update PPP

imhm committed:
update ppp

imhm committed:
Merge pull request #358 from imhm/final-bug-fixes

update ppp

JiawenLyu committed:
Merge pull request #357 from JiawenLyu/master

Update PPP

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

Jingming517 committed:
Updated user guide

Jingming517 committed:
Merge pull request #359 from Jingming517/master

Updated UG

Jingming517 committed:
Updated UG

LIU-YiFeng-1 committed:
update link to release

LIU-YiFeng-1 committed:
Merge pull request #360 from Jingming517/master

Updated UG

LIU-YiFeng-1 committed:
Merge pull request #361 from LIU-YiFeng-1/working_for_TP

update link to release

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

JiawenLyu committed:
Added PrioritizeCommand_SD.png

LIU-YiFeng-1 committed:
update no wifi and no local data exception

LIU-YiFeng-1 committed:
udpate exception

LIU-YiFeng-1 committed:
update Modulechecker filenotfoundexception

imhm committed:
update SD bugs

JiawenLyu committed:
Added PrintProgressCommand_SD.png

LIU-YiFeng-1 committed:
exception for filenotfound

LIU-YiFeng-1 committed:
update exception message

Jingming517 committed:
Changed theme in DG

LIU-YiFeng-1 committed:
Merge pull request #362 from LIU-YiFeng-1/working_for_TP

update no wifi and no local data exception

JiawenLyu committed:
Added PrintProgressCommand_SD.png

imhm committed:
Merge pull request #364 from imhm/final-bug-fixes

update SD bugs

JiawenLyu committed:
Merge pull request #363 from JiawenLyu/master

Amended SDs.png

LIU-YiFeng-1 committed:
update TOC

LIU-YiFeng-1 committed:
Merge pull request #365 from LIU-YiFeng-1/working_for_TP

update TOC

Jingming517 committed:
Merge pull request #366 from Jingming517/master

Changed theme in DG

Zhangyilin0203 committed:
update storage

LIU-YiFeng-1 committed:
update typo

LIU-YiFeng-1 committed:
Merge pull request #368 from LIU-YiFeng-1/working_for_TP

update typo

LIU-YiFeng-1 committed:
update typo

LIU-YiFeng-1 committed:
Merge pull request #369 from LIU-YiFeng-1/working_for_TP

update typo

Zhangyilin0203 committed:
Merge pull request #367 from Zhangyilin0203/bugs-fix

update storage

LIU-YiFeng-1 committed:
update UG typo

LIU-YiFeng-1 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into working_for_TP

Zhangyilin0203 committed:
update SD

Zhangyilin0203 committed:
Merge pull request #370 from Zhangyilin0203/bugs-fix

update SD

LIU-YiFeng-1 committed:
update on DG

Jingming517 committed:
timeline sd

LIU-YiFeng-1 committed:
Merge pull request #371 from LIU-YiFeng-1/working_for_TP

Working for tp

Jingming517 committed:
Merge pull request #372 from Jingming517/master

timeline sd

Jingming517 committed:
changed toc

Jingming517 committed:
Merge pull request #373 from Jingming517/master

changed toc

Zhangyilin0203 committed:
update storage

Zhangyilin0203 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-2/tp into bugs-fix-in-storage

Jingming517 committed:
Merge remote-tracking branch 'upstream/master'

Jingming517 committed:
toc

Zhangyilin0203 committed:
Merge pull request #375 from Zhangyilin0203/bugs-fix-in-storage

Bugs fix in storage

Jingming517 committed:
Merge pull request #374 from Jingming517/master

toc

Jingming517 committed:
toc

Jingming517 committed:
Merge pull request #376 from Jingming517/master

toc

LIU-YiFeng-1 committed:
update PPP

LIU-YiFeng-1 committed:
Merge pull request #377 from LIU-YiFeng-1/working_for_TP

update PPP

Jingming517 committed:
toc

Jingming517 committed:
Merge pull request #378 from Jingming517/master

toc

Jingming517 committed:
toc

Jingming517 committed:
Merge pull request #379 from Jingming517/master

toc

LIU-YiFeng-1 committed:
update PPP

LIU-YiFeng-1 committed:
Merge pull request #380 from LIU-YiFeng-1/working_for_TP

update PPP

Jingming517 committed:
sd

Jingming517 committed:
Merge pull request #381 from Jingming517/master

sd
